app/modules/sidebars/model/index.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export type Sidebar = string;
+
+export type SidebarsByName = Array<Sidebar>;
+
+export type SidebarsState = {
+  +byName: SidebarsByName,
```

Comment:
It's `byName` because everywhere else in the project `byId` is used for stuff like users, topics etc., but the Sidebars only have a name at the moment, so I named it `byName`. It's still just an array of strings now, but maybe this'll change in the future. But I can't see into the future, so for now I just followed the current convention.

Reply:
Current convention is that `byId` contains a mapping of `id` to `object`, though :P

I think the most future-proof way of modelling sidebars would be something like this (untested):

```
export type Sidebar = {
  +name: SidebarName,
  +isActive: boolean,
};

export type SidebarsByName = {
  +[sidebarName: SidebarName]: Sidebar,
};

export type SidebarsState = {
  +byName: SidebarsByName,
};
```